openshift-scope/README.md

```diff
@@ -0,0 +1,140 @@
+# Weave Scope on OpenShift
+This demo demonstrates how to install and run [Weave Scope](https://github.com/weaveworks/scope) on [OpenShift](https://www.openshift.com/).
+
+### What do you need to start?
+- Get an [OpenShift ready VM](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo.
+- Get the [demo](https://github.com/kinvolk/demo) repository.
+
+### Get the VM
+
+```
+git clone https://github.com/kinvolk/openshift-evangelists-vagrant-origin.git
+cd openshift-evangelists-vagrant-origin
+```
+
+Make sure the `Vagrantfile` has the following changes:
+- Fedora 25
+- 8GB mempory
+
+```
+VM_MEM = ENV['ORIGIN_VM_MEM'] || 8192 # Memory used for the VM
+...
+   config.vm.box = "fedora/25-cloud-base"
+```
+
+Now you can start the VM:
+
+```
+vagrant up --provider=virtualbpx
+```
+The building process could take several minutes depending of your machines because it is installing additional packages on top of a Fedora base image, and then fetching the last openshift sources and then build it. The final output should look like this:
+
+```
+==> default: You can now access OpenShift console on: https://10.2.2.2:8443/console
+==> default: To use OpenShift CLI, run:
+==> default: $ vagrant ssh
+==> default: $ sudo -i
+==> default: $ oc status
+==> default: $ oc whoami
+==> default: If you have the oc client library on your host, you can also login from your host.
+==> default: $ oc login https://10.2.2.2:8443
+```
+
+Install additional packages:
+```
+dnf install kernel-debug-modules-extra kernel-devel
```

Comment:
For some reason, I have to install also `kernel-modules-extra` to be able to load `sch_netem` so we should add it to the command.